app/controllers/admin/dashboard_controller.rb

```diff
@@ -2,6 +2,7 @@ class Admin::DashboardController < AdminController
   def index
     @site = Piwik::Site.load(6)
     @seo_info = @site.seo_info
+    @visits = getVisits()
```

Comment:
No need to have parenthesis for a method that doesn't have params

Reply:
Would you move the logic to the view?

In the future, I want things to be done asynchronously. (e.g. the page makes calls to the server and asks for content)

Reply:
Ah yeah they use `visits` instead of `visit_summary` as spec'd here: `https://github.com/matomo-org/piwik-ruby-api/blob/5fd3a74db5feafdc37089b8d2c3af475448f509c/lib/piwik/site.rb`

Reply:
Thanks, that worked